src/reducers/entities/integrations.js

```diff
@@ -89,17 +89,26 @@ function outgoingHooks(state = {}, action) {
 function commands(state = {}, action) {
     const nextState = {...state};
     switch (action.type) {
+    case IntegrationTypes.RECEIVED_COMMANDS:
     case IntegrationTypes.RECEIVED_CUSTOM_TEAM_COMMANDS: {
         for (const command of action.data) {
-            nextState[command.id] = command;
+            if (command.id) {
+                const id = command.id;
```

Comment:
Not a big deal and non-blocker, but weird to allocate a variable then immediately use it